pyfr/backends/base/generator.py

```diff
@@ -131,68 +131,70 @@ def render(self):
         pass
 
     def _deref_arg_view(self, arg):
+        ix = 'BLK_IDX + X_IDX'
+
         ptns = [
-            '{0}_v[{0}_vix[X_IDX]]',
-            r'{0}_v[{0}_vix[X_IDX] + SOA_SZ*(\1)]',
-            r'{0}_v[{0}_vix[X_IDX] + {0}_vrstri[X_IDX]*(\1) + SOA_SZ*(\2)]'
+            f'{arg.name}_v[{arg.name}_vix[{ix}]]',
```

Comment:
Revert to previous formatting approach.  This formats everything even though we only need one entry.